README.md

```diff
@@ -39,7 +39,7 @@ This instruction assumes that you want to create a single-node installation with
    ```
 3. Install DRAKVUF Sandbox stack:
    ```
-   sudo apt install redis-server
+   sudo apt install redis-server ntfs-3g
```

Comment:
Do we consider usermode profiles as something mandatory or optional?
Either way, we we should add it to  to .deb requirements / recommended.

Reply:
Well, it works without it, but you miss out on some cool features. I guess I'll move it to recommended + add info about it in readme.